File: Latihan 2/LatB_SepakBola (C++)/TeamSepakBola.cpp
class TeamSepakBola{

	private:
		//atribut dalam class team sepak bola
		string nama_team;
		string negara_asal;
		string tahun_berdiri;
		string nama_pemain;
		//atribut  tambahan
		string nama_kiper;

	public:
		//constructor
		TeamSepakBola(){
		}

		//prosedur dan fungsi untuk get & set dari atribut Nama Team
		void setNamaTeam(string nama_team){
			this->nama_team = nama_team;
		}
		
		string getNamaTeam(){
			return this->nama_team;
		}

		//prosedur dan fungsi untuk get & set dari atribut Negara Asal
		void setNegaraAsal(string negara_asal){
			this->negara_asal = negara_asal;
		}
		
		string getNegaraAsal(){
			return this->negara_asal;
		}

		//prosedur dan fungsi untuk get & set dari atribut Tahun Berdiri
		void setTahun(string tahun_berdiri){
			this->tahun_berdiri = tahun_berdiri;
		}
		
		string getTahun(){
			return this->tahun_berdiri;
		}

		//prosedur dan fungsi untuk get & set dari atribut Nama Pemain
		void setNamaPemain(string nama_pemain){
			this->nama_pemain = nama_pemain;
		}
		
		string getNamaPemain(){
			return this->nama_pemain;
		}

		//prosedur dan fungsi untuk get & set dari atribut Nama Kiper
		void setNamaKiper(string nama_kiper){
			this->nama_kiper= nama_kiper;
		}
		
		string getNamaKiper(){
			return this->nama_kiper;
		}
	//destructor
	~TeamSepakBola(){
	}

};

File: Latihan 2/LatB_SepakBola (Py)/TeamSepakBola.py
class TeamSepakBola():

	#atribut
	__nama_team = "";
	__negara_asal = "";
	__tahun_berdiri = "";
	__nama_pemain = "";
	#atribut  tambahan
	__nama_kiper = "";


	#constructor
	def __init__(self, nama_team = "", negara_asal = "", tahun_berdiri = "", 
				 nama_pemain = "", nama_kiper = ""):
		self.__nama_team = nama_team
		self.__negara_asal = negara_asal
		self.__tahun_berdiri = tahun_berdiri
		self.__nama_pemain = nama_pemain
		self.__nama_kiper = nama_kiper
		

	#get dan set untuk semua atribut
	def setNamaTeam(self, nama_team):
		self.__nama_team = nama_team

	def getNamaTeam(self):
		return self.__nama_team

	def setNegaraAsal(self, negara_asal):
		self.__negara_asal = negara_asal

	def getNegaraAsal(self):
		return self.__negara_asal

	def setTahun(self, tahun_berdiri):
		self.__tahun_berdiri = tahun_berdiri

	def getTahun(self):
		return self.__tahun_berdiri

	def setNamaPemain(self, nama_pemain):
		self.__nama_pemain = nama_pemain

	def getNamaPemain(self):
		return self.__nama_pemain

	def setNamaKiper(self, nama_kiper):
		self.__nama_kiper = nama_kiper

	def getNamaKiper(self):
		return self.__nama_kiper
